Content.Server/Shuttles/Commands/FTLDiskBurnerCommand.cs

```diff
@@ -0,0 +1,176 @@
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+
+public sealed class FTLDiskBurnerCommand : LocalizedCommands
+{
+    [Dependency] private readonly IEntityManager _entManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entSystemManager = default!;
+
+    public override string Command => "FTLdiskburner";
+
+    [ValidatePrototypeId<EntityPrototype>]
+    public const string CoordinatesDisk = "CoordinatesDisk";
+    public override void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length == 0)
+        {
+            shell.WriteError("Not enough arguments.");
+            return;
+        }
+
+
+        var player = shell.Player;
+
+        if (player == null)
+        {
+            shell.WriteLine("Only a player can run this command.");
+            return;
+        }
+
+        if (player.AttachedEntity == null)
+        {
+            shell.WriteLine("You don't have an entity to spawn a disk at.");
+            return;
+        }
+
+        EntityUid entity = player.AttachedEntity.Value;
+        var coords = _entManager.GetComponent<TransformComponent>(entity).Coordinates;
+
+        foreach (var destinations in args)
+        {
+            DebugTools.AssertNotNull(destinations);
+
+            // make sure destination is an id.
+            EntityUid dest;
+
+            if (_entManager.TryParseNetEntity(destinations, out var nullableDest))
+            {
+                DebugTools.AssertNotNull(nullableDest);
+
+                dest = (EntityUid) nullableDest;
+
+                // we need to go to a map, so check if the EntID is something else then try for its map
+                if (!_entManager.HasComponent<MapComponent>(dest))
+                {
+                    if (!_entManager.TryGetComponent<TransformComponent>(dest, out var entTransform))
+                    {
+                        shell.WriteLine(destinations + " has no Transform!");
+                        continue;
+                    }
+
+                    var mapSystem = _entSystemManager.GetEntitySystem<SharedMapSystem>();
+                    if (!mapSystem.TryGetMap(entTransform.MapID, out var mapDest))
+                    {
+                        shell.WriteLine(destinations + " has no map to FTL to!");
+                        continue;
+                    }
+
+                    DebugTools.AssertNotNull(mapDest);
+                    dest = mapDest!.Value; // explicit cast here should be fine since the previous if should catch it.
+                }
+
+                // find and verify the map is not somehow unusable.
+                if (!_entManager.TryGetComponent<MapComponent>(dest, out var mapComp)) // We have to check for a MapComponent here and above since we could have changed our dest entity.
+                {
+                    shell.WriteLine(destinations + " is somehow on map " + dest + " with no map component. What the fuck.");
```

Comment:
Leave the swearing to code comments, not things that get written out in the console. 😉